agent/util-scripts/pbench-register-tool-set

```diff
@@ -1,119 +1,153 @@
 #!/bin/bash
 # -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
 
-script_path=`dirname $0`
-script_name=`basename $0`
-pbench_bin="`cd ${script_path}/..; /bin/pwd`"
+script_path="$(dirname ${0})"
+script_name="$(basename ${0})"
+pbench_bin="$(realpath -e ${script_path}/..)"
 
 # source the base script
-. "$pbench_bin"/base
+. "${pbench_bin}"/base
 
-# The sole purpose of this script is to ensure a tool
-# of your choosing is used during the execution of a benchmark.
-# For a list of performance tools, look at the
-# $pbench_bin/tool-scripts directory.
+# The sole purpose of this script is to ensure a tool set of your choosing is
+# used during the execution of a benchmark on the optional list of remote
+# hostnames specified. For a list of available tool sets, use the --help
```

Comment:
"... available tools" perhaps? But --help does not seem to give that list.

Reply:
We don't register tools with `pbench-register-tool-set`, so I'll just get the available list with `--help`.

Reply:
Thanks; updated with output that now displays the available tool sets.